build64/Makefile

```diff
@@ -66,6 +213,8 @@ APP_OBJS = app/badram.o \
 
 OBJS = boot/startup.o boot/efisetup.o $(SYS_OBJS) $(LIB_OBJS) $(TST_OBJS) $(APP_OBJS)
 
+debug: clean memtest.debug memtest.efi
```

Comment:
Do you really want to do a clean build every time?

Reply:
Thank you for the remark. It was indeed not the intention to do a clean build every time. I just added it at the end to make sure that the whole script runs from the beginning without errors. I have removed it now